src/app/component/NavigationTabs/NavigationTabs.component.js

```diff
@@ -157,7 +157,8 @@ export class NavigationTabs extends NavigationAbstract {
 
         return (
             <footer block="NavigationTabs" mods={ { isHidden } }>
-                <nav block="NavigationTabs" elem="Nav">
+                { /* Id is required for product page to measure element`s height */ }
```

Comment:
Either reference the specific component here please or remove the comment at all, please